pymesync.py

```diff
@@ -0,0 +1,60 @@
+"""
+pymesync - Python TimeSync Module
+
+Allows for interactions with the TimeSync API
+- timesync.send_time(baseurl) -- Sends time to baseurl (TimeSync)
+"""
+import errno
+import json
+import requests
+
+
+class TimeSync(object):
+
+    def __init__(self, baseurl, user, password, auth_type):
```

Comment:
What if your auth type does not use a password? I'm thinking oauth or similar - maybe the auth required fields should be optional/required based on the auth type (for now, since all we have is password auth, that is effectively implemented, I just want ot make sure the implementation is not hard to modify for those things later)